pkg/cgroup/cgroup_stats_test.go

```diff
@@ -2,57 +2,124 @@ package cgroup
 
 import (
 	"os"
+	"path/filepath"
+	"runtime"
 	"testing"
 
+	"github.com/containerd/cgroups"
+	"github.com/containerd/cgroups/v3/cgroup1"
+	"github.com/opencontainers/runtime-spec/specs-go"
+
+	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+)
 
-	"github.com/sustainable-computing-io/kepler/pkg/utils"
+const (
+	defaultDirPerm = 0o755
```

Comment:
weird, I am having this error when executing `make` but I am able to run `make test` 
```
# make
make: command: Command not found
make: command: Command not found
GO111MODULE=off go get -d github.com/go-bindata/go-bindata/...
./hack/bindata.sh
go mod tidy -v
go mod vendor
./automation/presubmit-tests/gofmt.sh
Checking go format
pkg/cgroup/cgroup_stats_test.go:18:20: expected ';', found 'IDENT' o755
pkg/cgroup/cgroup_stats_test.go:26:1: expected 'IDENT', found 'var'
pkg/cgroup/cgroup_stats_test.go:28:1: expected 'IDENT', found 'type'
pkg/cgroup/cgroup_stats_test.go:28:1: missing constant value
pkg/cgroup/cgroup_stats_test.go:28:17: expected ';', found 'struct'

```

Reply:
This is very strange, I can run `./automation/presubmit-tests/gofmt.sh` with no problem....
It also ran in the CI with no problem....

@jichenjc did you do a new `git clone` or revert a old code?
Also, did you try to see those lines in `pkg/cgroup/cgroup_stats_test.go` that the go format is complaining?

Reply:
I will check more on this , this is an existing repo I do 
```
git fetch upstream pull/609/head:cgroupv1v2
```

to Pull your PR and do some test locally..

Reply:
Humm, I see. It might be worth trying to do a new `git clone` of kepler and then do `git fetch upstream pull/609/head:cgroupv1v2`. But I am not sure what is the problem....